pkg/resources/cruisecontrol/cruisecontrol.go

```diff
@@ -61,15 +61,30 @@ func (r *Reconciler) Reconcile(log logr.Logger) error {
 
 	log.V(1).Info("Reconciling")
 
-	for _, res := range []resources.ResourceWithLogs{
-		r.service,
-		r.configMap,
-		r.deployment,
-	} {
-		o := res(log)
-		err := k8sutil.Reconcile(log, r.Client, o, r.KafkaCluster)
+	if r.KafkaCluster.Spec.CruiseControlConfig.CruiseControlEndpoint == "" {
+
+		err := generateCCTopic(r.KafkaCluster)
```

Comment:
shouldn't we avoid calling this again once the status is `CruiseControlTopicReady`?

Reply:
Yes you are right will do.